kola/tests/bpf/bpf.go

```diff
@@ -59,7 +59,7 @@ func execsnoopTest(c cluster.TestCluster) {
 
 		// wait for the container and the `execsnoop` command to be correctly started before
 		// generating traffic.
-		if err := util.Retry(5, 2*time.Second, func() error {
+		if err := util.Retry(10, 2*time.Second, func() error {
```

Comment:
no clue about what's going on ? would be interesting to see a `systemd-analyze blame` but since every other tests are looking fine I guess it's really it's the pulling of the image ? 
We could try to pull a docker or a GHCR image to see if it changes something.